src/guide/scaling-up/sfc.md

```diff
@@ -54,16 +81,36 @@ That said, we do realize there are scenarios where SFCs can feel like overkill.
 
 Vue SFC is a framework-specific file format and must be pre-compiled by [@vue/compiler-sfc](https://github.com/vuejs/core/tree/main/packages/compiler-sfc) into standard JavaScript and CSS. A compiled SFC is a standard JavaScript (ES) module - which means with proper build setup you can import an SFC like a module:
 
-```js
-import MyComponent from './MyComponent.vue'
+<div class="options-api">
```

Comment:
I don't think we should change this second example. The current example is intended to show importing a `.vue` file into a plain `.js` file. It's supposed to illustrate how `.vue` files are just ES modules. Showing a `.vue` importing a `.vue` doesn't quite make the same point about compatibility with normal JavaScript.

The existing example is valid for both the Options API and the Composition API. The `components` option is common to both.